server/controllers/yelpController.js

```diff
@@ -1,3 +1,8 @@
+// CONNECTS TO YELP RESTFUL API
+// CONVERTING TO YELP_GRAPH_QL
+
```

Comment:
**To do:** Delete this file in future pull request after talking to @Rodolfoguzman25 about tests